### ISSUE TITLE
Can't use BMM commands in Paper 1.19.3

### ISSUE THREAD

**Issue by NyaomiDEV** [MiraculixxT/bluemap-marker#7]
```
[10:09:37 ERROR]: Caught previously unhandled exception :
[10:09:37 ERROR]: Paper Async Command Builder Thread Pool - 5
java.lang.NoSuchMethodError: 'boolean net.minecraft.commands.CommandListenerWrapper.k()'
    at de.miraculixx.bmm.MarkerCommand.mainCommand$lambda$38$lambda$37$lambda$36$lambda$31(MarkerCommand.kt:111) ~[bmm-paper-1.3.0.jar:?]
    at com.mojang.brigadier.tree.CommandNode.canUse(CommandNode.java:79) ~[paper-1.19.3.jar:git-Paper-446]
    at net.minecraft.commands.Commands.fillUsableCommands(Commands.java:477) ~[?:?]
    at net.minecraft.commands.Commands.fillUsableCommands(Commands.java:512) ~[?:?]
    at net.minecraft.commands.Commands.fillUsableCommands(Commands.java:512) ~[?:?]
    at net.minecraft.commands.Commands.sendAsync(Commands.java:429) ~[?:?]
    at net.minecraft.commands.Commands.lambda$sendCommands$5(Commands.java:408) ~[?:?]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
    at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

This also has the side effect of bugging out the entire autocompletion, for other commands too.

```
This server is running Paper version git-Paper-446 (MC: 1.19.3) (Implementing API version 1.19.3-R0.1-SNAPSHOT) (Git: 4e99466)
```

**Comment by MiraculixxT** [comment]
That happens, if you use the non Legacy Paper Version of BMM in versions older then 1.19

**Comment by MiraculixxT** [comment]
Fixed in latest version for 1.18.x - 1.19.x
